### ISSUE TITLE
Should chunked data keep current chunk shape by default when saving?

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#2613]
If a hyperspy signal already contains chunked dask array data, it will currently not automatically save the array with those chunk shapes unless specified in the `chunks` argument to `s.save()`. Should it? I think it would be the expected behaviour to keep the chunk shape.

It would be good to clarify this in the documentation, since people typically save large files.

**Example code:**
```python
import hyperspy.api as hs
import dask.array as da

data = da.random.random((10000, 3000))
data = data.rechunk((100, 3000))
s = hs.signals.Signal1D(data)
s = s.as_lazy()
s.data # chunks: (100, 3000)

s.save('test.hspy')
s2 = hs.load('test.hspy', lazy=True)
s2.data # chunks: (40, 3000)

s.save('test2.hspy', chunks = (100, 3000))
s3 = hs.load('test2.hspy', lazy=True)
s3.data # chunks: (100, 3000)
```

**Comment by ericpre** [comment]
Indeed, this should be an easy fix, are you going to make a PR?

**Comment by ericpre** [comment]
Fixed in #2629.
